repos.d/slackware.yaml

```diff
@@ -0,0 +1,48 @@
+###########################################################################
+# Slackware
+###########################################################################
+- name: slackware-current
```

Comment:
I'm using underscores in internal repo names, let's stick to this "rule" here as well.